tests/test_dtr.py

```diff
@@ -168,6 +169,27 @@ def test_read_8(get_fn):
     assert eq(xyz_ref[:, ::2, :], xyz)
 
 
+def test_read_as_traj_1(get_fn):
+    # Test the default read_as_traj and compare against reference trajectory in dcd format
+    fn_dtr = get_fn("frame0.dtr")
+    fn_dcd = get_fn("frame0.dcd")
+    fn_pdb = get_fn("frame0.pdb")
+    top = _parse_topology(fn_pdb)
```

Comment:
Do we need to load the topology in for testing? Admittedly, I'm basing a lot of my ideas off `test_xtc` and `test_hdf5` (the formats I use most) but in those cases we don't need to load topology for testing. Just considering if this may be unneeded bloat for testing purposes (or maybe topology DTR testing needs to be a separate test)